drafts/draft-ietf-ippm-ioam-data-integrity.xml

```diff
@@ -602,93 +654,107 @@
-          inserted at all, then the encapsulating node MUST NOT use an Integrity
-          Protected Option-Type.</t>
-
-          <t>A transit node generates a signature over the Signature field and
-          the hash of IOAM-Data-Fields it has inserted, i.e.,
-          sign(Signature || hash(IOAM-Data-Fields)). IOAM-Data-Fields modified
-          in-place by the transit node MUST be excluded from the signature
-          (e.g., the POT Cumulative field). The signature is stored in the Signature
-          field of the IOAM Integrity Protection header. Important note: if the
-          transit node does not insert IOAM-Data-Fields (e.g., it only modifies
-          IOAM-Data-Fields in-place, or does nothing), then the transit node
-          MUST NOT generate a signature and MUST NOT update the Signature field.</t>
-
-          <t>The decapsulating node (aka the Validator) is responsible for the
-          integrity verification of the IOAM-Data-Fields collected. Serving as
-          the Validator, the decapsulating node MUST NOT generate a signature
-          based on IOAM-Data-Fields it has inserted, if any, and therefore MUST
-          NOT update the Signature field. In order to check the IOAM-Data-Fields
-          integrity, the Validator verifies the signature chain by iteratively
-          following the same procedure in the reverse order (i.e., from the last
-          node to the encapsulating node). For each node, the Validator applies
-          the corresponding key (see <xref target="SymmetricMethod" /> or
-          <xref target="AsymmetricMethod" /> depending on the approach, i.e.,
-          symmetric or asymmetric) to the signature (initially, the Signature
-          field) to retrieve both the previous signature and the hash of the
-          IOAM-Data-Fields signed by the node. The retrieved hash is then
-          compared to the computed hash of the corresponding IOAM-Data-Fields
-          inserted by the node. As a result, the Validator can detect if the
-          IOAM-Data-Fields integrity is intact or was altered. The validation is
-          trivial in some cases (e.g., with POT Type-0 or E2E Option-Types),
-          where only the encapsulating node generates a signature, as specified
-          by the method described above in this section. For other cases where
-          transit nodes also generate a signature (e.g., with Trace Option-Types),
-          node-ids MUST be included in IOAM-Data-Fields. Details on how the
-          mapping between node-ids and keys is implemented on the Validator are
-          outside the scope of this document.</t>
-        </list>
-      </t>
-
-      <section anchor="SymmetricMethod" title="Symmetric key based signature">
+    <section title="Space optimized integrity protection method">
+      <t>This section defines a space optimized method for integrity protection
+      of IOAM-Data-Fields. With this method, the signature consumes as few bytes
```

Comment:
*This section defines a space optimized method for integrity protection
      of IOAM-Data-Fields. With this method, the signature consumes as few bytes
      as possible and is carried only once per packet. In case of performance
      concerns, the method can be applied to a subset of the traffic by using
      sampling of data.*
      
      Consider changing to:
      
       This section defines a method for integrity protection
      of IOAM-Data-Fields. In case of performance
      concerns, the method can be applied to a subset of the traffic by using
      sampling of data.